client/packages/common/src/intl/utils/IntlUtils.ts

```diff
@@ -213,35 +217,32 @@ const getFullName = (
   }
 };
 
-interface GetNumberColumnLabelProps {
+type InputVerb = 'received' | 'in-stock' | 'available' | 'issued';
+
+export interface GetVaccineItemLabelProps {
   t: TypedTFunction<LocaleKey>;
-  displayInDoses: boolean;
   displayInPack?: boolean;
-  itemUnit?: string | null;
-  columnName?: string;
+  unitName?: string | null;
+  inputVerb?: InputVerb;
 }
 
-const getNumberColumnLabel = ({
+const getVaxItemLabel = ({
```

Comment:
Honestly... probably don't need this anymore since translations are {{unit}} *area*... 🤷🏻